packages/app/src/components/pages/Home/styles.css

```diff
@@ -50,3 +52,33 @@
 .detail-font-size {
   font-size: 1rem;
 }
+
+.green * {
+  color: #128B3E !important;
+  font-weight: bold;
+}
+
+.stage-dropdown {
+  margin-bottom: 0px;
+  margin-left: 5px;
+  width: auto;
+}
+
+.stage-dropdown:focus {
+  border-color: #128B3E;
+  /* had to update the url to change from purple using path fill, because milligram hardcodes the color into the svg image */
```

Comment:
Huh good catch. That's pretty weird on Milligram's part.

Reply:
Thanks- Jim was a big help!